backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/llvm/objcexport/BlockPointerSupport.kt

```diff
@@ -136,9 +136,11 @@ private val BlockPointerBridge.nameSuffix: String
 internal class BlockAdapterToFunctionGenerator(val objCExportCodeGenerator: ObjCExportCodeGenerator) {
     private val codegen get() = objCExportCodeGenerator.codegen
 
+    private val kRefSharedHolderType = LLVMGetTypeByName(codegen.runtime.llvmModule, "class.KRefSharedHolder")!!
```

Comment:
Hmm, indeed `class.`?

Reply:
That's the name of the LLVM type produced by clang for `class KRefSharedHolder`.